_posts/atoms/2018-09-07-icons.html

```diff
@@ -8,14 +8,222 @@
              and [more](https://fontawesome.com/how-to-use/on-the-web/styling/).
 ---
 
-`Add`:
-%i.fas.fa-plus-circle.text-primary
-Add
+%h3 Icon Modifiers
+
+Rotate with `fa-spin`:
+%i.fas.fa-sync-alt.fa-spin
+%br
+
+Rotate on click with `fa-spin`:
```

Comment:
```suggestion
Rotate on click with `rotate-on-click`:
```

Reply:
This `rotate-on-click` class doesn't exist in OBS, it's just here to provide an example on how this could be achieved.

Reply:
Ah, ok, I thought it was a typo. I'll dismiss it